### ISSUE TITLE
Any Option to remove the 'Attached Text (text/x.cucumber.log+plain)' from Cucumber Offline HTML report?

### ISSUE THREAD

**Issue by pabitra-qa** [cucumber/react-components#335]
In Cucumber Published report, the interface is really clean but the offline report says 'Attached Text (text/x.cucumber.log+plain)' for each scenario.log(). Any easy way to make the offline report look like the published report?

**Published**
<img width="782" alt="image" src="https://github.com/cucumber/cucumber-jvm/assets/109291193/22ae6303-647a-4e4f-a387-e417f41f9e56">

**Offline**
<img width="857" alt="image" src="https://github.com/cucumber/cucumber-jvm/assets/109291193/1fac5eff-d626-497c-8c48-5708f4647e77">

**Comment by mpkorstanje** [comment]
@pabitra-qa the report is constructed out of [cucumber/react-components](https://github.com/cucumber/react-components). Improvements are best made there.

**Comment by davidjgoss** [comment]
We can do something with this, yeah. The original point of the collapsed-by-default state of attachments was to prevent images (e.g. screenshots) just dominating the page. But we could be smarter about textual attachments especially if they're only a line or two.

**Comment by pabitra-qa** [comment]
@davidjgoss yeah, it would be really nice if we can match the published report look in offline report. The published one is really neat and clean. Thank you!

**Comment by pabitra-qa** [comment]
Hi @davidjgoss, Any updates on this?
